WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackFeaturesRemovalCoordinator.swift

```diff
@@ -107,6 +107,22 @@ class JetpackFeaturesRemovalCoordinator {
         return formatter.date(from: dateString)
     }
 
+    /// Used to determine if the Jetpack features should be removed based on the removal phase.
+    static func shouldRemoveJetpackFeatures() -> Bool {
```

Comment:
@hassaanelgarem heads up that I already introduced this helper in https://github.com/wordpress-mobile/WordPress-iOS/pull/19691. This is most likely the reason why this PR has conflicts with `WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackFeaturesRemovalCoordinator.swift`. The conflict should be relatively easy to address but let me know if I can help out with this.

Reply:
Thanks for the heads up @fluiddot!
I actually cherrypicked this change from your PR to limit the conflicts as much as possible. I fixed the conflict in c0ad1588c910efb29f617cc7b8cf333902832d00 👍